modules/core/client/views/careers.client.view.html

```diff
@@ -10,15 +10,32 @@ <h1 class="header center orange-text">Careers</h1>
 			    <h5 class="header col s12 light">Available Positions</h5>
 			</div>
 
-			<div class="row center">
-		  		<a id="contact-us-button" class="btn-large waves-effect waves-light orange" ng-click="openCareerApplication();">Apply Now</a>
-			</div>
-
 		</div>
 
 	</div>
 
-	<div class="lightbox ng-hide" ng-show="careerApplicationVisible">
+  <div class="row left">
+    <ul class="left jobsList">
+      <li class="jobList-item" ng-repeat="job in Jobs">
+        <div class="row">
+          <h5 class="col s12 light">{{job}}</h5>
+        </div>
+        <div class="row">
+          <span class="col s12 light">Description of Job goes here blah blah blah blah blah b lbkajasdfkjhasdkjfhaskjdhfksajdhfaksjdhfkajshfdksajhdfajksdfhasjkdfhajksdhfsdjkfhajkshdf
```

Comment:
Yeah, I just modeled this after the to-do list. Hugh and Daniel are going to connect it to the database on Thursday I believe.